src/actions/boardColumns/actionCreators.js

```diff
@@ -0,0 +1,56 @@
+import types from './types';
+
+export const createNewColumn = () => ({
+  type: types.CREATE_NEW_COLUMN
+})
+
+export const createNewColumnSuccess = (response) => ({
+  type: types.CREATE_NEW_COLUMN_SUCCESS,
+  payload: response
+})
+
+export const createNewColumnFailed = (error) => ({
+  type: types.CREATE_NEW_COLUMN_FAILED,
+  payload: error
+})
+
+export const getBoardColumns = () => ({
+  type: types.GET_BOARD_COLUMNS
+})
+
+export const getBoardColumnsSuccess = (response) => ({
+  type: types.GET_BOARD_COLUMNS_SUCCESS,
+  payload: response
+})
+
+export const getBoardColumnsFailed = (response) => ({
+  type: types.GET_BOARD_COLUMNS_FAILED,
+  payload: response
+})
```

Comment:
Get rid of this `payload: response` in all actionCreators and try to put understandable data to payload, for example: 

```
{
  ....
  payload: {
    boards
  }
}
```